mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -142,6 +142,8 @@ def __init__(self, app_name, ad):
         self._counter = None
         self._lock = threading.Lock()
         self._event_client = None
+        self.verbose_logging = False   # Logging RPC response with full string or truncated by given length
+        self._max_rpc_return_value_length = 1024
```

Comment:
this should remain a const?

Reply:
Well, it could be modify by user, not really need a const.